cfgov/cfgov/settings/base.py

```diff
@@ -130,6 +129,7 @@
 STATICFILES_FINDERS = (
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'compressor.finders.CompressorFinder',
```

Comment:
Is this still needed? I thought it was just for using the compress tag which we wouldn't use since we use our own compression.